internal/action/show.go

```diff
@@ -306,7 +306,12 @@ func showSafeContent(sec gopass.Secret) string {
 		}
 	}
 
-	sb.WriteString(sec.Body())
+	for _, l := range strings.Split(sec.Body(), "\n") {
+		if strings.HasPrefix(l, "otpauth://") {
+			continue
+		}
```

Comment:
This isn't behaving exactly like the rest of the `safecontent` since this is currently just suppressing the lines. To be consistent, I think it should be putting `randAsterisk()` to replace what comes after `otpauth://` IMO.

Also, if the otp entry is a key, this won't hide it, we currently support the keys `otpauth`, `hotp` and `totp`:
https://github.com/gopasspw/gopass/blob/01b6ff1dde17dcd139b5d8c8ca3c52808cd79c60/pkg/otp/otp.go#L30-L47

So you might want to add these to the `isUnsafeKey` function to cover all the cases, both when the otp URL is a key and is in the `Body`.